output.tf

```diff
@@ -1,3 +1,7 @@
 output "key_name" {
-  value = "${aws_key_pair.default.key_name}"
+  value = "${aws_key_pair.non_empty.key_name}"
```

Comment:
rewrite like this `${element(compact(aws_key_pair_non_empty.*.key_name, aws_key_pair_empty.*.key_name), 0)}`

Reply:
empty output with:
```terraform
output "key_name" {
  value = "${element(compact(aws_key_pair.imported.*.key_name, aws_key_pair.generated.*.key_name), 0)}"
}
```
non-empty output with:
```terraform
output "key_name" {
  value = "${element(compact(concat(aws_key_pair.imported.*.key_name, aws_key_pair.generated.*.key_name)), 0)}"
}
```